packages/in_app_purchase/in_app_purchase_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0
+* **BREAKING CHANGE**: Removes `launchPriceChangeConfirmationFlow` from `InAppPurchaseAndroidPlatform` as it is deprecated by Android.
```

Comment:
If the underlying method is deprecated, can we deprecate this instead of removing it outright? Or does it not make sense in the context of having migrated to v5?

Reply:
V5 calls make use of `ProductDetails` objects, where V4 call use `SkuDetails`. As `launchPriceChangeConfirmationFlow` requires `SkuDetails`, it is incompatible with the V5 calls.